Makefile

```diff
@@ -187,7 +187,7 @@ print-cover-opt:
 coverage:
 	cover ${COVER_OPTS} -test
 
-COVER_REPORT_OPTS ?= -select_re ^lib/
+COVER_REPORT_OPTS ?= -select_re ^(lib|script)/
```

Comment:
same here

Reply:
yes, I excluded that parts again for now and will leave it to https://github.com/os-autoinst/openQA/pull/2967